server/webapp/WEB-INF/rails.new/webpack/views/elastic_profiles/elastic_profile_modal_body.js.msx

```diff
@@ -42,7 +42,7 @@ const ElasticProfileModalBody = {
       return vnode.attrs.pluginInfos().mapPluginInfos((pluginInfo) => {
         return {
           id:   pluginInfo.id(),
-          text: pluginInfo.id()
+          text: pluginInfo.about().name() || pluginInfo.id()
```

Comment:
can we add the version as well in the describe? something such as _"Docker Elastic Agent Plugin [v0.5]"_